htdocs/core/class/commonobject.class.php

```diff
@@ -4129,6 +4130,9 @@ public function fetchObjectLinked($sourceid = null, $sourcetype = '', $targetid
 									$object = new $classname($this->db);
 									$ret = $object->fetch($objectid);
 									if ($ret >= 0) {
+										if (method_exists($object, 'fetch_thirdparty')) {
+											$object->fetch_thirdparty();
```

Comment:
If you add the fetch_thirdparty here, it means all the code calling the fetchObjectLinked will generate an extra load of data even for caller that don't need it. Please move the fetch_thirdparty at a higher level (so the caller of fetchObjectLinked or the caller of caller, ...)